docs/HowToGuides/GettingStarted.md

```diff
@@ -68,15 +68,17 @@ toolchain as a one-off, there are a couple of differences:
      If you plan on contributing regularly, cloning over SSH provides a better
      experience. After you've [uploaded your SSH keys to GitHub][]:
      ```sh
-     git clone git@github.com:apple/swift.git .
+     git clone git@github.com:apple/swift.git swift
```

Comment:
The dot may be superfluous but it's not incorrect. I would leave it in.

Reply:
The dot is what makes it incorrect. Unless we change to the `swift` directory first.